### ISSUE TITLE
Can't login with default credentials 

### ISSUE THREAD

**Issue by davidrutland** [hay-kot/mealie#276]
**Describe the bug**
Created mealie using Docker compose and attempted to log in using the default credentials

Username: changeme@email.com
Password: MyPassword

 Receive a 'Could Not Validate Credentials' message

**Device Information (please complete the following information):**
 - Raspbian on Raspberry Pi4B
 - Deployment:docker-compose
 - Browser:Firefox

**Comment by natpuch** [comment]
This is linked to #274.
A (temporary) fix, if your database is empty, is to delete the database (in `/app/data/db/) and running the `init_db.py` script (for this go to `/app` and run `python mealie/db/init_db.py`).

**Comment by TheRealVira** [comment]
For anybody stumbling upon this post and wondering why it says "invalid credentials"... it's because the default e-mail changed from `changeme@email.com` to `changeme@example.com`.

**Comment by mikkelsiggaard** [comment]
> For anybody stumbling upon this post and wondering why it says "invalid credentials"... it's because the default e-mail changed from `changeme@email.com` to `changeme@example.com`.

For me it was the other way around. 
changeme@email.com was correct.

**Comment by michael-genson** [comment]
If the default credentials are using changeme@email.com, you're likely on an older version:
https://nightly.mealie.io/documentation/getting-started/installation/installation-checklist/#migrating-from-other-v1-versions

**Comment by jmricker** [comment]
I think I had the same problem using what I thought was the newest version. I went to docker hub (https://hub.docker.com/r/hkotel/mealie/tags) and used the latest tag but now I'm looking at it, it shows that was updated 2 years ago. Is there a better way to get the most recent docker container?

**Comment by coffee-mage** [comment]
https://nightly.mealie.io/documentation/getting-started/installation/sqlite/

TLDR: 
image: ghcr.io/mealie-recipes/mealie:v1.0.0-RC1.1

**Comment by jmricker** [comment]
yep thats what I switched too. I skimmed through the instructions and went straight to docker hub so that is how I ended up with an older version. After reading through the upgrade information that @michael-genson provided, I switched to that image.

**Comment by mikkelsiggaard** [comment]
Is it on purpose that the "latest"-tag on Dockerhub isn't the most recent version? 
(Because of RC status?)

**Comment by ballinger** [comment]
having the same problem on 2.3 i suppose?
```
INFO     2024-12-01T09:21:06 - [IP:0] 401 Unauthorized "POST /api/auth/token HTTP/1.1"
INFO     2024-12-01T09:21:10 - [127.0.0.1:37478] 200 OK "GET /api/app/about HTTP/1.1"
INFO     2024-12-01T09:21:40 - [127.0.0.1:59654] 200 OK "GET /api/app/about HTTP/1.1"
ERROR    2024-12-01T09:22:02 - Incorrect username or password from IP
ERROR    2024-12-01T09:22:02 - Incorrect username or password from IP
INFO     2024-12-01T09:22:02 - [IP:0] 401 Unauthorized "POST /api/auth/token HTTP/1.1"
INFO     2024-12-01T09:22:11 - [127.0.0.1:39530] 200 OK "GET /api/app/about HTTP/1.1"
```

**Comment by ballinger** [comment]
tried postgress with the same "credentials error" in the ui flashing up. did a normal docker install and using yunohost reverse proxy for the ngnix stuff. guess this looks more like a yunohost error?
`2024/12/01 13:52:04 [error] 1690385#1690385: *438744 SSL_do_handshake() failed (SSL: error:0A00010B:SSL routines::wrong version number) while SSL handshaking to upstream, client: IP, server: food.domain.org, request: "GET /sw.js HTTP/2.0", upstream: "https://127.0.0.1:9925/sw.js", host: "food.domain.org"`

**Comment by BrookeDot** [comment]
Just noting I got this error when I didn't have SMTP set, which is required per the documentation. It seems failure to set SMTP prevents the app from starting and it may fail silently on login.

**Comment by Kuchenpirat** [comment]
You don't need to have SMTP configured for mealie to start. In fact our default compose files do not have SMTP configured. 

Could you please point out to where in the docs it is marked as required.
I would like to change this to be more clear then.
